packages/domain/lib/exceptions/validation_exception.dart

```diff
@@ -0,0 +1,15 @@
+enum ValidationExceptionStatus {
+  ok,
+  empty,
+  notCorrect,
```

Comment:
technically, all failed validation cases are not correct. so we should to rename the enum item

Reply:
Fixed.